lib/sbuf.c

```diff
@@ -0,0 +1,85 @@
+#include <zebra.h>
```

Comment:
sbuf.c needs a gnu style header.  We cannot accept this until we have a license that we can use :)